packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -3,130 +3,184 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-panel-background-color: Specifies the background color of the component.
+ * @prop --calcite-panel-border-color: Specifies the border color of the component.
+ * @prop --calcite-panel-description-color: Specifies the color of the component's description.
+ * @prop --calcite-panel-fab-z-index: Specifies the component fab's z-index.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
  * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
+ * @prop --calcite-panel-header-background-color: Specifies the component header's background color.
  * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+ * @prop --calcite-panel-header-color: Specifies the component header's  color.
+ * @prop --calcite-panel-header-z-index: Specifies the component header's z-index.
```

Comment:
Could we make a `--calcite-panel-z-index` instead? I'm curious what the use case would be for a user to need to change the layer position of only the panel header.

Reply:
I prefer the specificity here IMO. If we say `--calcite-panel-z-index` users may think its the whole panel's z-index.

They also have different values as is. One uses the header zindex and the other uses the sticky zindex (FAB).